include/collection.h

```diff
@@ -918,7 +922,8 @@ class Collection {
                                   const size_t& max_filter_by_candidates = DEFAULT_FILTER_BY_CANDIDATES,
                                   bool rerank_hybrid_matches = false,
                                   bool validate_field_names = true,
-                                  bool enable_analytics = true) const;
+                                  bool enable_analytics = true,
+                                  std::string analytics_tags="") const;
```

Comment:
Need to support only a single tag, so change to `analytics_tag` (see explanation in the other review comment)